xcell/cls/cl.py

```diff
@@ -244,6 +244,20 @@ def get_cl_file(self):
                 cl_cov_12_cp *= correction
                 cl_cov_22_cp *= correction
 
+            if (mapper1.wf is not None) or (mapper2.wf is not None):
+                # Applies a given window function to
+                # the cl's
+                wf_a = mapper1.wf(ell)
+                wf_b = mapper2.wf(ell)
+                correction = wf_a*wf_b
+                # Apply correction to all Cl's
+                cl /= correction
+                cl_cp /= correction
```

Comment:
I think this can't be right, since `cl` and `cl_cp` are sampled at different ells.

Reply:
you are right it should only be applied to cl, no to cl_cp

Reply:
removed wf from cl.py and moved to mapper_base. See above